twitch/connection.go

```diff
@@ -57,6 +58,12 @@ func (irc *IRC) connectIRC() error {
 	// TODO: define function that stores message to db
 	c.OnPrivateMessage(func(msg v2.PrivateMessage) {
 		irc.PersistChat(msg)
+		fmt.Println(len(irc.msgQueue))
+		select {
+		case msg := <-irc.msgQueue:
```

Comment:
If you want to get all the messages in the queue your going to need a for loop. . . Not sure when this func is called. . .

Reply:
You can range over a channel, the for loop will auto break when the channel is closed && empty.

Reply:
good call